misaligned.c

```diff
@@ -1,21 +1,49 @@
 #include <stdio.h>
 #include <assert.h>
+#include <string.h>
+#include "misaligned.h"
+#include "misaligned_test.h"
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+void formatColorPairString(colorPair color_pair_info, char *color_pair_buffer)
+{
+    memset(color_pair_buffer, 0, MAX_COLOR_PAIR_STRING_SIZE);
+    sprintf(color_pair_buffer, "%d\t|%s\t|%s", color_pair_info.colorCode, color_pair_info.majorColor, color_pair_info.minorColor);

```

Comment:
Yes. It is aligned correctly while printing. Since I have used \t, I am not able to make the expected output string similar to the printed output. Should I use some regex to make it similar in expected string and printed output?